config/nrfconnect/chip-module/Kconfig.features

```
@@ -304,4 +304,13 @@ choice CHIP_LAST_FABRIC_REMOVED_ACTION
 
 endchoice
 
+config CHIP_LAST_FABRIC_REMOVED_SCHEDULE_TIME
```

Minor: the Kconfig name sounds weird and it's hard to guess the meaning without reading description. How about small change?
```suggestion
config CHIP_LAST_FABRIC_REMOVED_ACTION_DELAY
```